### ISSUE TITLE
明确bert中 layer的shape

### ISSUE THREAD

**Issue by reporter-law** [bojone/bert4keras#492]
提问时请尽可能提供如下信息：

### 基本信息
- 你使用的**操作系统**: windows
- 你使用的**Python**版本: python3.7
- 你使用的**Tensorflow**版本: tensorflow 1.15.4+nv
- 你使用的**Keras**版本: 2.2.4
- 你使用的**bert4keras**版本: 0.11.3
- 你使用纯**keras**还是**tf.keras**: keras
- 你加载的**预训练模型**:roformer_v2

### 核心代码
```python
 print(bert.output)
```

### 输出信息
```
Tensor("bertTransformer-5-FeedForward-Norm/truediv:0", shape=(?, ?, 384), dtype=float32)
```

### 自我尝试
需求：希望明确shape,主要是maxlen,形如Tensor("bertTransformer-5-FeedForward-Norm/truediv:0", shape=(?, 300, 384), dtype=float32)

**Comment by i4never** [comment]
是根据batch中的max确定的、为啥一定要明确？

**Comment by reporter-law** [comment]
因为想对maxlen层处理，比如concatenate(axis=1)

**Comment by bojone** [comment]
如果你的“明确”是指固定，那么可以在`build_transformer_model`的时候传入`sequence_length=xxx`

**Comment by reporter-law** [comment]
> 如果你的“明确”是指固定，那么可以在`build_transformer_model`的时候传入`sequence_length=xxx`


是的，就是固定，感谢苏神的回复！！！
